doc/whats_new.rst

```diff
@@ -456,6 +456,12 @@ Bug fixes
       <https://github.com/scikit-learn/scikit-learn/pull/7159>`_)
       by `Yichuan Liu <https://github.com/yl565>`_.
 
+    - Add ``_pairwise`` to :class:`OneVsOneClassifier` and
+      :class:`OneVsRestClassifier`.  Also moves ``_safe_split``
```

Comment:
It might read more like "Cross-validation of OvR and OvO models now works with precomputed kernels". No mention of `_safe_split`, as @raghavrv says.